### DISCUSSION TITLE
Find a hidden word

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#584] [Galaxy Trivia] [answered]
<img width="257" alt="image" src="https://user-images.githubusercontent.com/5120290/232841113-09589014-796f-4864-b480-aa0f09e8b660.png">

**Comment by alejandroch1202** [comment, marked as answer]
E - R - O - T - I - C - A ?

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/alejandroch1202?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Comment by thuongtruong1009** [comment]
The hidden word is "START"
